letsencrypt-auto

```diff
@@ -19,13 +19,20 @@ else
   SUDO=
 fi
 
-for arg in "$@" ; do 
+for arg in "$@" ; do
   # This first clause is redundant with the third, but hedging on portability
   if [ "$arg" = "-v" ] || [ "$arg" = "--verbose" ] || echo "$arg" | grep -E -- "-v+$" ; then
     VERBOSE=1
   fi
 done
 
+if [ -f /etc/debian_version ] ; then
+    # squeeze needs Python 2.6, `python2` doesn't exist
+    if grep -q '^6\.0' /etc/debian_version ; then
```

Comment:
I think it's out of scope for this PR. Moreover, proposed suggestion is much more likely to break. Let's keep it simple for now, and just fix squeeze.